index.js

```diff
@@ -209,36 +209,41 @@ module.exports = function (opts) {
         done(err)
       } else {
         // Render all individual content.
-        var contentFiles = minimatch.match(filesKeys, opts.pattern, {matchBase: true})
-        async.map(contentFiles, processFile, function (err) {
-          if (err) {
-            done(err)
-          } else {
-            // Render the content within the layouts.
-            async.map(contentFiles, renderContent, function (err) {
-              if (err) {
-                done(err)
-              } else {
-                // Delete the layout data.
-                async.map(layouts, deleteFile, function (err) {
-                  if (err) {
-                    done(err)
-                  } else {
-                    // Now rename all the files.
-                    async.map(Object.keys(files), renameFile, function (err) {
-                      if (err) {
-                        done(err)
-                      } else {
-                        // Now delete auxiliary metadata from files.
-                        async.map(Object.keys(files), deleteAuxiliaryMetadata, done)
-                      }
-                    })
-                  }
-                })
-              }
-            })
-          }
-        })
+        try {
+          var contentFiles = minimatch.match(filesKeys, opts.pattern, {matchBase: true})
```

Comment:
Rather than just wrapping the whole thing in a try/catch, should we split it up into just the above line, and then the individual async function calls?